libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/stumblerthread/scanners/ScanManager.java

```diff
@@ -176,10 +171,8 @@ public synchronized void startScanning(Context ctx) {
         mLocationChangeSensor.start();
 
         if (mMotionSensor == null) {
-            if (mDetectMotionReceiver != null) {
-                LocalBroadcastManager.getInstance(mAppContext).registerReceiver(mDetectMotionReceiver,
-                        new IntentFilter(MotionSensor.ACTION_USER_MOTION_DETECTED));
-            }
+            LocalBroadcastManager.getInstance(mAppContext).registerReceiver(mDetectMotionReceiver,
```

Comment:
Why is the null check gone here?  We should probably have a testcase that captures the case where `mDetectMotionReceiver` is null.